manual/src/main/asciidoc/user-guide/collectors.adoc

```diff
@@ -472,11 +472,28 @@ This feature installs a default `etc/org.apache.karaf.decanter.collector.socket.
 
 # Protocol tcp(default) or udp
 #protocol=tcp
+
+# Dataformat json(default) : 
+# - The dataformat has to match the unmarshaller property in the component annotation
+# - Example : property = Marshaller.SERVICE_KEY_DATAFORMAT + "=json"
+unmarshaller.target=(dataformat=json)
 ----
 
 * the `port` property contains the port number where the network socket collector is listening
 * the `workers` property contains the number of worker thread the socket collector is using for connection
 * the `protocol` property contains the protocol used by the collector for transferring data with the client
+* the `dataformat` property contains the unmarshaller used by the collector to transform the data
```

Comment:
This is not accurate as the property is `unmarshaller.target`, not `dataformat`.